files-community/Files
# Feature: Updated the edit tags button in the Details Pane

**Resolved / Related Issues**
   Related #12946

**Validation**
How did you test these changes?
   1. Open app and navigate to a folder containing a tagged item
   2. Enable the preview pane
   3. Select the tagged item

**Screenshots (optional)**
![image](https://github.com/files-community/Files/assets/102259289/7798415f-60ff-4407-a15b-5aa35fb9a5ea)
